specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,228 @@
+MonochromaticOverlayPresenter
+===
+
+# Background
+
+The `MonochromaticOverlayPresenter` API in this spec is a Xaml Framework Element that lets you "draw" its background by
+pointing at another element; the rendering of that other element becomes the background of
+the Grid.
```

Comment:
I think we should remove the notion of "background" since we got rid of the content, so the whole background vs content issue is moot. Talking too much about "background" makes you think that this is about custom backgrounds, when really it's about re-rendering another element.

```suggestion
The `MonochromaticOverlayPresenter` API is a Xaml Framework Element that renders the portion of another element at the same position (typically beneath it in the Z-order), while also applying a monochromatic colorization effect. This allows the element to appear to be an overlay on top of the other element.
```